controllers/applicationvolumereplication_controller.go

```diff
@@ -485,15 +502,14 @@ func (r *ApplicationVolumeReplicationReconciler) findManifestWork(
 	return nil, nil
 }
 
-func (r *ApplicationVolumeReplicationReconciler) deleteExistingManfiestWork(
-	avr *ramendrv1alpha1.ApplicationVolumeReplication,
+func (a *AVRInstance) deleteExistingManfiestWork(
```

Comment:
(nit) Can you please change the name of this function from "deleteExistingManfiestWork" to "deleteExistingManifestWork" ? Its a small typo in the name, but helps in searching.

Reply:
good catch.  Thanks